docs/learn/tools.md

```diff
@@ -410,6 +410,47 @@ The core concept to understand here is that the `suggest_author` tool's descript
 
 This is why the "beginner" recommendation and "advanced" recommendations call the `suggest_author` tool with authors befitting the reading level of each call.
 
+
+## Pre-Made Tools
+
+Mirascope provides several pre-made tools to help you get started quickly:
+
+| Tool | Primary Use | Key Features | Characteristics |
+|------|-------------|--------------|-----------------|
+| `HTTPX` | Advanced HTTP Requests | • Full HTTP method support (GET/POST/PUT/DELETE)<br>• Custom header support<br>• File upload/download<br>• Form data handling | • Async support (AsyncHTTPX)<br>• Configurable timeouts<br>• Comprehensive error handling<br>• Redirect control |
```

Comment:
we need to also include these tools in the API reference and then link from here to the API reference so users can easily see full details of each pre-made tool